XenAdmin/Dialogs/ScheduledSnapshots/ScheduledSnapshotsDialog.cs

```diff
@@ -446,6 +446,23 @@ private void dataGridViewRunHistory_CellClick(object sender, DataGridViewCellEve
             }
         }
 
+        private void dataGridViewRunHistory_ColumnHeaderMouseClick(object sender, DataGridViewCellMouseEventArgs e)
+        {
+            if (e.ColumnIndex != ColumnImage.Index)
+                return;

```

Comment:
We also need to reset the `ColumnImage.HeaderCell.SortGlyphDirection` (to `SortOrder.None`) here when the other column headers are clicked, so that the sort arrow is being cleared from the Image column when we are sorting on other columns